app/renderer/components/Session/CloudProviderSelector.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import _ from 'lodash';
+import { Modal, Row, Col, Button } from 'antd';
+import CloudProviders from './CloudProviders';
+import SessionStyles from './Session.css';
+
+export default class CloudProviderSelector extends Component {
```

Comment:
Is i18n added automatically to props? I thought you need to wrap the exported class into i18n HOC before exporting it to make that possible

Reply:
The props get passed from the root to all subcomponents.

Reply:
This is how `i18n` gets passed into the component:

```js
<CloudProviderSelector {...this.props} />
```